.github/workflows/pullrequest.yml

```diff
@@ -158,8 +158,11 @@ jobs:
       - name: Run codegen
         if: steps.generated-files-cache.outputs.cache-hit != 'true' || env.SKIP_GENERATED_CACHE == 'true'
         run: |
+          set -euo pipefail
           node --version
-          tar zcvf generated_files.tar.gz $(./scripts/ci/get-files-touched-by.sh yarn codegen --skip-cache | xargs realpath --relative-to $(pwd))
+          touched_files="$(./scripts/ci/get-files-touched-by.sh yarn codegen --skip-cache)"
```

Comment:
Indeed, a lot. Here I'd rather revert any changes than adding even more to this PR.

The `set -o pipefail` is pretty nice, though, to prevent obscure "missing argument" errors